### DISCUSSION TITLE
<Flex> on:click or onclick -example ?

### DISCUSSION THREAD

**Discussion by germgerm** [svelteuidev/svelteui#488] [General]
how to wire the click event handler for <Flex> items ?
nothing I have tried works

**Comment by BeeMargarida** [comment]
Flex does not have on:click handler. One option is to use a button, you can use the `UnstyledButton` if you don't want the default button style.
Now, I think perhaps we should allow the on:click handler, since it's a div under the hood, for now I'll treat this as a bug, and the workaround can be the solution I mentioned above.
